tests/services/test_file_freezer.py

```
@@ -152,23 +150,23 @@ def test_freeze_file_already_in_cold(self, db_session, file_inventory_factory, s
         """Test that freezing fails if file is already in cold storage."""
         inv = file_inventory_factory(path="/tmp/cold/file.txt", storage_type=StorageType.COLD)
         monitored_path = db_session.get(MonitoredPath, inv.path_id)
-        
+
         success, error, cold_path = FileFreezer.freeze_file(
```

Ruff (0.15.7)

[warning] Unpacked variable `cold_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)